CMakeLists.txt

```diff
@@ -271,6 +291,7 @@ if(LIBDEFLATE_BUILD_SHARED_LIB)
                           PUBLIC_HEADER libdeflate.h
                           C_VISIBILITY_PRESET hidden
                           SOVERSION 0)
+    configure_framework(libdeflate_shared)
```

Comment:
In the [previous response](https://github.com/ebiggers/libdeflate/pull/421#r1987659390), I oversaw the fact that one can build frameworks for macOS, so setting up the properties for 
shared-libs do make sense...